dbt/include/athena/macros/utils/timestamps.sql

```diff
@@ -3,7 +3,7 @@
 %}
 
 {% macro athena__current_timestamp() -%}
-    cast(now() as timestamp)
+    cast(now() as timestamp(6))
```

Comment:
It will break with Athena Engine v2 no ?

Reply:
it will, but users using v2 they can overwrite this macro, and also "iceberg" merge is only available in v3 so I guess is fine.

Reply:
Yes but `athena__current_timestamp` is not only for merge & snapshot right ?